loader/src/main/resources/META-INF/defaultfmlconfig.toml

```diff
@@ -19,3 +19,8 @@ earlyWindowControl = true
 #Max threads for early initialization parallelism,  -1 is based on processor count
 maxThreads = -1
 
+# Define dependency overrides below
+# Dependency overrides can be used to forcibly remove a dependency constraint from a mod or to force a mod to load AFTER another mod
+# Using dependency overrides can cause issues. Use at your own risk.
+# Example dependency override for the mod with the id 'targetMod': dependency constraints (incompatibility clauses or restrictive version ranges) against mod 'dep1' are removed, and the mod will now load after the mod 'dep2'
+# dependencyOverrides.targetMod = ["-dep1", "+dep2"]
```

Comment:
Unfortunately this file is not actually used. When I delete the `config/fml.toml`, it is NightConfig's correction system that is used. We should document this in the generated config file somehow.

Reply:
Ugh, you're right, and night configs have no way of storing root comments...

Reply:
I'll add an empty `dependencyOverrides = {}`

Reply:
How are you testing?

Reply:
Okay nevermind, it was hidden in the middle of the file. 😓
But it should be at the end, why is it not...

Reply:
I have no idea what ordering fml uses